cloudferrylib/base/migration.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+"""Migration of objects from SRC to DST clouds."""
+
+
+from cloudferrylib.utils import utils
+
+
+CLOUD = 'cloud'
+SRC, DST = 'src', 'dst'
+
+
+class Migration(object):
```

Comment:
This seem to only be used in cinder migration, doesn't need to be moved to a `base` package, it's not globally base